conjur/api/api.py

```diff
@@ -13,8 +13,8 @@
 from datetime import datetime, timedelta
 
 # Internals
-from conjur.endpoints import ConjurEndpoint
-from conjur.http_wrapper import HttpVerb, invoke_endpoint
+from conjur.api.endpoints import ConjurEndpoint
```

Comment:
GENERAL NOTE, is there a way that you can manually test the initialization of the Client. Ensure you can still perform basic actions (like list, set variable)?

Reply:
@tzheleznyak confirmed the client is still working